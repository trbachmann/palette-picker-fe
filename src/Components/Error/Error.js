import React from 'react';

const Error = () => {
  return(
    <div>
      <h2>Sorry! Something went wrong :( Please try again later!</h2>
    </div>
  )
};

export default Error;