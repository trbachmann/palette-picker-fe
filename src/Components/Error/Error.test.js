import React from 'react';
import { shallow } from 'enzyme';
import Error from './Error';

describe('Error', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Error />);
    expect(wrapper).toMatchSnapshot();
  });
});