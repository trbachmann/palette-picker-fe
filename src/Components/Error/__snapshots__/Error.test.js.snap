// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the snapshot 1`] = `
<div>
  <h2>
    Sorry! Something went wrong :( Please try again later!
  </h2>
</div>
`;
